sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/models/KubernetesCluster.java

```diff
@@ -463,15 +472,50 @@
             WithCreate withSystemAssignedManagedServiceIdentity();
         }
 
-//        /** The stage of the Kubernetes cluster definition allowing to specify Kubernetes Role-Based Access Control. */
-//        interface WithRBAC {
-//            /**
-//             * Enables Kubernetes Role-Based Access Control.
-//             *
-//             * @return the next stage
-//             */
-//            WithCreate withRBACEnabled();
-//        }
+        /** The stage of the Kubernetes cluster definition allowing to specify Kubernetes Role-Based Access Control. */
+        interface WithRBAC {
+            /**
+             * Disables Kubernetes Role-Based Access Control.
+             *
+             * @return the next stage
+             */
+            WithCreate disableKubernetesRbac();
```

Comment:
What happens if KubernetesRbac is disabled? Does it mean user can only access the cluster using azure rbac?

Reply:
I assume the admin/user KubeConfig should still work? They are the basics, unless be closed via `disableLocalAccouts`.

Azure RBAC requires Kubernetes RBAC. Cluster creation fails if this Kubernetes RBAC disabled.

Reply:
> AKS-managed Azure AD integration can't be disabled
> Changing a AKS-managed Azure AD integrated cluster to legacy AAD is not supported
> Clusters without Kubernetes RBAC enabled aren't supported for AKS-managed Azure AD integration